### DISCUSSION TITLE
please help me resolve this error

### DISCUSSION THREAD

**Discussion by ashanwe** [vercel/next.js#72262] [Help]
### Summary

I installed Next 15 & getting this error when i run it.

Hydration failed because the server rendered HTML didn't match the client. As a result this tree will be regenerated on the client. This can happen if a SSR-ed Client Component used

**Comment by icyJoseph** [comment]
Show us the code and errors you get, the error overlay contains a lot of hints already... This can be any number of things, so we need more input from you.

**Reply by ashanwe** [reply]
![Screenshot (54)](https://github.com/user-attachments/assets/4fceb444-355e-42bc-9caf-26a19095db97)

**Reply by icyJoseph** [reply]
Looks like you are applying that className, hydrated, client side? Could you show us how you do that?

**Reply by ashanwe** [reply]
That's the problem. I didn't apply the className. I just only installed Next15 and run it

**Reply by icyJoseph** [reply]
You could've included all of this information initially.

What node is getting that className? Is this a fresh project? Or are you upgrading?

**Reply by ashanwe** [reply]
fresh project. If I install another one, get same error

**Reply by icyJoseph** [reply]
One more thing, does it happens on Incognito Mode as well?
